styles/fonts.css

```diff
@@ -0,0 +1,45 @@
+
+@font-face {
+  font-family: "Montserrat";
+  font-style: normal;
+  font-weight: 300;
+  font-display: swap;
+  src: url("/fonts/montserrat-v15-latin-300.eot");
+  src: local(""),
+    url("/fonts/montserrat-v15-latin-300.eot?#iefix")
+      format("embedded-opentype"),
+    url("/fonts/montserrat-v15-latin-300.woff2") format("woff2"),
+    url("/fonts/montserrat-v15-latin-300.woff") format("woff"),
+    url("/fonts/montserrat-v15-latin-300.ttf") format("truetype"),
+    url("/fonts/montserrat-v15-latin-300.svg#Montserrat") format("svg");
+}
+
+@font-face {
+  font-family: "Montserrat";
+  font-style: normal;
+  font-weight: 500;
+  font-display: swap;
+  src: url("/fonts/montserrat-v15-latin-500.eot");
+  src: local(""),
+    url("/fonts/montserrat-v15-latin-500.eot?#iefix")
+      format("embedded-opentype"),
+    url("/fonts/montserrat-v15-latin-500.woff2") format("woff2"),
+    url("/fonts/montserrat-v15-latin-500.woff") format("woff"),
+    url("/fonts/montserrat-v15-latin-500.ttf") format("truetype"),
+    url("/fonts/montserrat-v15-latin-500.svg#Montserrat") format("svg");
```

Comment:
Were these ``@font-face`` at-rules auto-generated? I'm asking these because it has more than one src, and I think that just `woff` and `woff2` formats are necessary to be optimized. 

By doing so, we'll be able to remove the other formats like `eot` and `svg` from the static resources and decrease the fetching time.

Reply:
The font-face are generated by: https://google-webfonts-helper.herokuapp.com/fonts

The outher formats is to support outhers browser, but I think we can remove the `eot` and `svg`.

Reply:
Can you send the link of the browser support for these formats?

Reply:
So, I will remove the older browes fonts and just let the `woff` and `woff2` formats.

Browser that have support to woff: Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+
Browser that have support to woff2: Chrome 26+, Opera 23+, Firefox 39+

Reply:
``eot`` doesn't seem to be supported in a lot of browsers

https://caniuse.com/?search=eot

Reply:
We can leave ``ttf``, ``woff`` and ``woff2``

https://caniuse.com/?search=ttf 
https://caniuse.com/?search=woff
https://caniuse.com/?search=woff2

Reply:
@LauraBeatris 

So I will let just the ´woff´ and ´woff2´ formats, it's ok?

Reply:
Would be also good to `ttf`` according to caniuse.com/?search=ttf, for instance, it supports older versions of Safari

``woff``
![image](https://user-images.githubusercontent.com/48022589/99815434-0f94de00-2b29-11eb-96cc-c8db2dbc6f6a.png)

``ttf``
![image](https://user-images.githubusercontent.com/48022589/99815480-20ddea80-2b29-11eb-840b-348b33e7722c.png)